modules/system/defaults/ActivityMonitor.nix

```diff
@@ -18,7 +18,7 @@ with lib;
             105: Active Processes
             106: Inactive Processes
             107: Windowed Processes
-          Default is 100.
+          Default is null.
```

Comment:
The documentation option usually describe what the macOS default is, not configured is't very helpful 😄.